src/content/tools/diagnostic-messages.md

```diff
@@ -10710,7 +10710,7 @@ abstract class C {
 
 ### invalid_null_aware_operator
 
-_The receiver can't be null because of short-circuiting, so the null-aware
+_The receiver can't be 'null' because of short-circuiting, so the null-aware
```

Comment:
Backticks for code font?

Reply:
This comes from the SDK, but I believe diagnostic messages use single quotes on purpose. I forget exactly why, but I think it is due to how IDEs surface the messages and currently can't or don't apply Markdown formatting.